cmd/app/link.go

```diff
@@ -115,7 +115,7 @@ func LinkCommandRunE(ctx context.Context, clients *shared.ClientFactory, app *ty
 // explaining how to link apps, in case the user declines.
 func LinkAppHeaderSection(ctx context.Context, clients *shared.ClientFactory, shouldConfirm bool) {
 	var secondaryText = []string{
-		"Add an existing app created on app settings",
+		"Add an existing app from app settings",
```

Comment:
note: App Settings now displays Slack CLI created apps as read-only, un-clickable. So, I figured we should update this text to not be restricted to "created on app settings" because the app may be created by the CLI.